src/server/connection.rs

```diff
@@ -1236,31 +1258,6 @@ impl Connection {
         }
     }
 
-    fn get_supported_resolutions(display_idx: usize) -> Option<SupportedResolutions> {
```

Comment:
You added this in your PR https://github.com/rustdesk/rustdesk/pull/6628. So why?

Reply:
In #6628 I skipped supported resolutions on wayland as it doesn't work on wayland and it was making calls to `get_capturables` on loop and hence user prompts on loop.

But in this PR I have changed `get_capturables` so that it makes only one user prompt request no matter how many calls it gets. https://github.com/rustdesk/rustdesk/commit/eff2490081c48c4f251847de5024cb8d0e2a8c1e
So reverting #6628 enables the future possibility of supporting `supported resolutions` on wayland with only one user prompt. Currently, it is working the same with or without it.

Reply:
Good job.